package.json

```diff
@@ -37,22 +37,22 @@
     "@angular/platform-browser": "~13.0.0",
     "@angular/platform-browser-dynamic": "~13.0.0",
     "@angular/router": "~13.0.0",
-    "@capacitor/android": "^3.2.3",
-    "@capacitor/app": "^1.0.3",
-    "@capacitor/cli": "^3.2.3",
-    "@capacitor/clipboard": "^1.0.6",
-    "@capacitor/core": "^3.2.3",
-    "@capacitor/device": "^1.0.3",
-    "@capacitor/local-notifications": "^1.0.6",
-    "@capacitor/push-notifications": "^1.0.4",
-    "@capacitor/splash-screen": "^1.1.2",
-    "@ionic-native/core": "^5.33.1",
-    "@ionic-native/device": "^5.33.1",
-    "@ionic-native/http": "^5.33.1",
-    "@ionic-native/media": "^5.33.1",
-    "@ionic-native/status-bar": "^5.33.1",
-    "@ionic/angular": "^6.0.1",
-    "@ionic/pwa-elements": "^3.0.2",
+    "@capacitor/android": "^3.4.0",
```

Comment:
All package updates are minor version bumps only, so not expected to introduce any breaking changes